test/topology_experimental_raft/test_mv_tablets.py

```diff
@@ -73,3 +74,113 @@ async def test_tablet_mv_simple_6node(manager: ManagerClient):
     # We used SYNCHRONOUS_UPDATES=TRUE, so the view should be updated:
     assert [(3,2)] == list(await cql.run_async("SELECT * FROM test.tv WHERE c=3"))
     await cql.run_async("DROP KEYSPACE test")
+
+# Convenience function to open a connection to Alternator usable by the
+# AWS SDK. When more than one test needs it, we can move this function to
+# a separate library file. Until then, let's just have it in front of the
+# only test that needs it.
+import boto3
+import botocore
+alternator_config = {
+    'alternator_port': 8000,
+    'alternator_write_isolation': 'only_rmw_uses_lwt',
+}
+def get_alternator(ip):
+    url = f"http://{ip}:{alternator_config['alternator_port']}"
+    return boto3.resource('dynamodb', endpoint_url=url,
+        region_name='us-east-1',
+        aws_access_key_id='alternator',
+        aws_secret_access_key='secret_pass',
+        config=botocore.client.Config(
+            retries={"max_attempts": 0},
+            read_timeout=300)
+    )
+
+# Alternator convenience function for fetching the entire result set of a
+# query into an array of items.
+def full_query(table, ConsistentRead=True, **kwargs):
+    response = table.query(ConsistentRead=ConsistentRead, **kwargs)
+    items = response['Items']
+    while 'LastEvaluatedKey' in response:
+        response = table.query(ExclusiveStartKey=response['LastEvaluatedKey'],
+            ConsistentRead=ConsistentRead, **kwargs)
+        items.extend(response['Items'])
+    return items
+
+async def inject_error_on(manager, error_name, servers):
+    errs = [manager.api.enable_injection(s.ip_addr, error_name, False) for s in servers]
+    await asyncio.gather(*errs)
+
+# FIXME: boto3 is NOT async. So this test is not async. We could use
+# the aioboto3 library to write a really asynchronous test, or implement
+# an async wrapper to the boto3 functions ourselves (e.g., run them in a
+# separate thread) ourselves.
+@pytest.mark.asyncio
+async def test_tablet_alternator_lsi_consistency(manager: ManagerClient):
```

Comment:
Is there any point running this on release mode?
In release mode the injection will not do anything so it is kind of pointless to run it (unless you are counting on accidentally catching this bug without the delay.

Reply:
I am not strongly opinionated about this so maybe we can mark this test (and probably should do it for all other tests that uses injection) somehow in a follow up.

Reply:
@eliransin I don't know that injection is ignored in release mode. Why?
It means that **any** test that uses injection cannot be run on release mode - just on dev mode. Are we fine with that? CC @avikivity  

The way I would do this in the cql-pytest framework is to *skip* a test when it cannot be done on this build mode - the inject function can detect it's not supported and skip the calling test - with no need for any separate marks. I can do this here too but am still trying to figure out the conventions of the topology test framework. Note that my absolute last preference would be to manually list this test in some yaml file of excluded tests as I've seen people doing - I think this approach is a mistake.

Reply:
@nyh This is how you skip a test in release mode:
```
@pytest.mark.asyncio
@skip_mode('release', 'error injections are not supported in release mode')
async def test_table_dropped_during_streaming(manager: ManagerClient):
```

If you skip the test before invoking the injection API we save some run time because we avoid cluster bootstrap.

Reply:
Thanks, I'll do this, though I don't love the need to remember to add this marker.
I still have an uneasy feeling about the principle though - just a few days we had a big discussion about needed more "testability", more injection points and so on that will make it easier to write reliable and shorter tests. But if we say that these injection points don't work in release mode, isn't this a problem? Doesn't it mean we should **avoid** using injection points in tests, or at least **most** tests, so that we'll still have most testings be able to run on "release mode" (which is the most important binary to be testing!!!!)
CC @avikivity I wonder what your thoughts on this are.

Reply:
I don't see a problem. We're testing the algorithm here.

There's a chance that there's an additional bug that is hidden by both dev and debug modes and is exposed by release mode, but the probability is low, and together with the low probability of the injected failure happening in production, I think it's okay.

Reply:
I do this skip.